setup.cfg

```diff
@@ -102,6 +101,7 @@ install_requires =
     # Deprecation tagging
     deprecated
     # Redis cache
+    redis[hiredis]
     django-redis
```

Comment:
Note that in django 4 we dont require the `django-redis` package as its now implemented via `django.core.cache.backends.redis.RedisCache`

Reply:
We may need a bit more extensive testing for the switch. I recommend we do that separately and not as part of this upgrade. For more context on `django-redis` [see](https://github.com/jazzband/django-redis/issues/585#issuecomment-1002051473).